.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,32 +29,15 @@ or from https://gitmoji.dev/
 
 
 ## Related issue/s
-
-<!-- Link pull request to an issue
-  SEE https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
-
-- resolves ITISFoundation/osparc-issues#428
-- fixes #26
--->
+<!-- LINK to other issues and add prefix `closes`, `fixes`, `resolves`-->
 
 
 ## How to test
 
 <!-- Give REVIEWERS some hits or code snippets on how could this be tested -->
 
-## Dev-ops checklist
+## Dev-ops
 
-- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))
```

Comment:
why you remove this link? that is actually helpful

Reply:
It is not removed. rather moved. but ok, i will bring thatone back